projects/sb-user-management/src/lib/user-count.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
```

Comment:
@poojakarma 
Please move these service files in the user-count folder

Reply:
Resolved